index.js

```diff
@@ -67,6 +67,35 @@ function toggleColor(element) {
     }
 }
 
+const headerText = document.querySelector('h1#header')
+
+headerText.addEventListener('click', () => {
+    toggleColor(headerText)
+})
+
+
+
+
 /***** Deliverable 2 *****/
+//1. find element
+const likeButton = document.querySelector('button.like-button')
+
+
+//2.
+ likeButton.addEventListener('click', () => {
+     console.log('worked')
+ })
+
+
+ const likeButton = document.querySelector('p.likes')
+ const likeCount = document.querySelector('p.likes')
+
+ count = 1000
+ likeButton.addEventListener('click', function(element){
+     count += 1;
+     likeCount.innerHTML = count + " Likes" ;
+ })
+ //.split & parseInt
+
 
 /***** Deliverable 3 *****/
```

Comment:
Let's lay this one out!

1. Find the sightingForm on the page and save it to a variable
2. add an event listener to the form element, listen for a `submit` event.
3. The submit event needs to be applied to an entire form element, not the submit button.
4. Create the `handleFormSubmission` callback function, which should be triggered when the form submits.

- `handleFormSubmission` steps:
1. since we are submitting a form, we need to prevent the default action of the form, which is to make a post request and redirect to a specific URL. We do not want the page to refresh, we want to prevent that action and stay on the current page.
2. save all the data the user typed in from the form into descriptive variables.
3. assemble an object with key value pairs descriptive of the data they contain.
4. Pass in the animal data object to the `renderAnimalSightingPost` function
```javascript
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);
function handleFormSubmission(e){
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species: species,
        videoLink: videoLink,
        photo: photo,
        description: description
    }
    renderAnimalSightingPost(newData);
}
sightingForm.addEventListener("submit", handleFormSubmission)
```
With ES6 notation
```javascript
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);
const handleFormSubmission = e => {
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species,
        videoLink,
        photo,
        description
    }
    e.target.reset()
    renderAnimalSightingPost(newData);
}
sightingForm.addEventListener("submit", handleFormSubmission)
```